js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -11,7 +12,10 @@ export function getSearchParams(keys: string[]): string[] {
 }
 
 export function getDemoInputs(model: ModelData, keys: (number | string)[]): any[] {
```

Comment:
I'm confused about the role of this function. I see it's called from concrete Widgets' `onMount`, but there is also a similar filtering in `WidgetWrapper`:

https://github.com/huggingface/hub-docs/blob/557b21cbca1a5aa5e49173eab84662270b7670d8/js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte#L41

It would help if the responsibility of preparing/filtering the examples & chosing which one to display first was done all in the same place.

Reply:
related to #1021 ?

Reply:
yes!
Let's go for #1021 and rely on the `WidgetWrapper` to filter examples then?
Wdyt @mishig25 ?

Reply:
yes! ^^

Reply:
handled by https://github.com/huggingface/hub-docs/pull/1023